transforms/pyproject.toml

```
@@ -1,7 +1,7 @@
 [project]
 name = "data_prep_toolkit_transforms"
 version = "1.1.7.dev0"
```

May want to increment this to dev4

```suggestion
version = "1.1.7.dev4"
```